src/extractor/extractor.cpp

```diff
@@ -531,7 +533,9 @@ std::
         });
 
     tbb::filter_t<SharedBuffer, std::shared_ptr<ExtractionRelationContainer>> buffer_relation_cache(
-        tbb::filter::parallel, [&](const SharedBuffer buffer) {
+        tbb::filter::parallel,
+        // NOLINTNEXTLINE(performance-unnecessary-value-param)
+        [&](const SharedBuffer buffer) {
```

Comment:
Was not sure if here we should pass it by reference because in case of multithreading often things are intentionally copied, so decided to just suppress it.

Reply:
Do you think I should fix it here too? IMO it is still a bit risky, but won't give any significant benefits.

Reply:
I'd suggest leaving it as it is, unless you want to dig into the TBB internals to confirm.